cylc/rose/stem.py

```diff
@@ -259,15 +269,17 @@ def _get_base_dir(self, item):
            * project
         """
 
-        ret_code, output, stderr = self.popen.run('fcm', 'loc-layout', item)
+        ret_code, output, stderr = self.popen.run(
+            'fcm', 'loc-layout', src_tree)
         if ret_code != 0:
-            raise ProjectNotFoundException(item, stderr)
+            raise ProjectNotFoundException(src_tree, stderr)
 
         ret = {}
         for line in output.splitlines():
             if ":" not in line:
                 continue
             key, value = line.split(":", 1)
+
             if key and value:
```

Comment:
@dpmatthews  - This is splitting lines from `fcm loc-layout`. Will it ever be false?